service/src/main/kotlin/com/jongmin/upbit/quotation/candles/UpbitCandleService.kt

```diff
@@ -0,0 +1,54 @@
+package com.jongmin.upbit.quotation.candles
+
+interface UpbitCandleService {
+
+    /**
+     * 분봉 캔들 조회
+     *
+     * @param
+     * market: 마켓 코드
+     * to: 마지막 캔들시각 (yyyy-MM-dd'T'HH;mm:ss'Z' or yyyy-MM-dd HH:mm:ss)
+     * count: 캔들 개수
```

Comment:
We don't need it now, but let's add a value that can be used later in the comments!
```
캔들 개수(최대 200개까지 요청 가능)
```